src/test/java/com/google/api/generator/engine/writer/JavaWriterVisitorTest.java

```diff
@@ -205,6 +208,103 @@ public void writeVariableExpr_scopedStaticFinalDecl() {
     assertEquals(writerVisitor.write(), "public static final boolean x");
   }
 
+  @Test
+  public void writeBlockComment_basic() {
+    String content = "this is a test comment";
+    BlockComment blockComment = BlockComment.builder().setComment(content).build();
+    String expected = "/** this is a test comment */\n";
+    blockComment.accept(writerVisitor);
+    assertEquals(writerVisitor.write(), expected);
+  }
+  // TODO(xiaozhenliu): add comment escaper for BlockComment/JavaDocComment classes and update unit
```

Comment:
As discussed in the last review, the escaping-related unit tests should go in BlockComment or JavaDocComment tests, not in JavaWriterVisitorTest. Could we please update this TODO?

Reply:
updated, and also put a TODO in JavaDocCommentTest, thanks.